core/src/ibc/client_02/manager.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Kodebox, Inc.
+// Copyright 202- Kodebox, Inc.
```

Comment:
Typo here.

Reply:
Please become the first reviewer of your PR.
You can reduce these types of mistakes if you review your PR first.

Reply:
Sorry. I was kinda rushing this PR because I should be quickly shared to you for interface design.